packages/codemod/test/data-state.test.mjs

```
@@ -0,0 +1,226 @@
+import assert from 'node:assert/strict'
+import { describe, it } from 'node:test'
```

ESLint

[error] Import from `vitest` instead of `node:test`

(test/no-import-node-test)